.travis.yml

```diff
@@ -164,26 +159,8 @@ script:
         singlenode-kafka -g kafka
     fi
   - |
-    if [[ -v HIVE_TESTS ]]; then
-      presto-hive-hadoop2/bin/run_hive_tests.sh
-    fi
-  - |
-    if [[ -v HIVE_TESTS && -v HIVE_TESTS_AWS_ACCESS_KEY_ID ]]; then
-      env AWS_ACCESS_KEY_ID="${HIVE_TESTS_AWS_ACCESS_KEY_ID}" \
-        AWS_SECRET_ACCESS_KEY="${HIVE_TESTS_AWS_SECRET_ACCESS_KEY}" \
-        S3_BUCKET_ENDPOINT="${S3_TESTS_BUCKET_ENDPOINT}" \
-        S3_BUCKET="${S3_TESTS_BUCKET}" \
-        presto-hive-hadoop2/bin/run_hive_s3_tests.sh
-    fi
-  - |
-    if [[ -v HIVE_TESTS && -v HIVE_TESTS_AWS_ACCESS_KEY_ID ]]; then
-      env AWS_ACCESS_KEY_ID="${HIVE_TESTS_AWS_ACCESS_KEY_ID}" \
-        AWS_SECRET_ACCESS_KEY="${HIVE_TESTS_AWS_SECRET_ACCESS_KEY}" \
-        ./mvnw test $MAVEN_SKIP_CHECKS_AND_DOCS -B -pl presto-hive -P test-hive-glue
-    fi
-  - |
-    if [[ -v MODULES_WITH_EXTRA_TESTS ]]; then
-        ./bin/run_tests_for_modules.sh "${MODULES_WITH_EXTRA_TESTS}"
+    if [[ -v TEST_SPECIFIC_MODULES ]]; then
+        ./bin/run_tests_for_modules.sh "${TEST_SPECIFIC_MODULES}"
```

Comment:
This should be `MODULES_WITH_EXTRA_TESTS` here (`Extract HIVE_TESTS out of .travis.yml` commit)